cypress/integration/spec.js

```diff
@@ -36,4 +36,16 @@ describe('Article search', () =>{
     cy.enter()
     articlePage.title().should('have.text', "Catt")
   })
+
+  it('back button should take us to search page with results', () => {
+    searchPage.search("cattle")
+    searchPage.results().first()
+    cy.enter()
+    cy.downArrow()
+    cy.enter()
+    articlePage.title().should('have.text', "Cattle")
+    cy.clickCloseButton()
+    searchPage.results().first()
+      .children().first().next().should('have.class', 'img')
```

Comment:
how do we tell having the class img belongs to search page with results?

Reply:
I'm sorry, I don't understand the question

Reply:
I'm not very sure, let's resolve this.